Meta/Lagom/Fuzzers/FuzzZip.cpp

```diff
@@ -8,7 +8,7 @@
 #include <stddef.h>
 #include <stdint.h>
 
-extern "C" int LLVMFuzzerTestOneInput(uint8_t const* data, size_t size)
+extern "C" int LLVMFuzzerTestOneInput(u8 const* data, size_t size)
```

Comment:
This one seems not great either. the declaration of `LLVMFuzzerTestOneInput` comes from LibFuzzer. it's not our function. we should make the definition match the llvm header exactly.

Reply:
`LLVMFuzzerTestOneInput` is declared with `extern "C"`, so realistically it doesn't matter what the argument types are. We actually do have a few fuzzers that already use `u8` instead of `uint8_t`.

Reply:
I added `static_assert` to AK that uint8_t is in fact the same as u8.